usrsctplib/netinet/sctp_output.c

```diff
@@ -13853,7 +13853,7 @@ sctp_lower_sosend(struct socket *so,
 		SCTP_TCB_UNLOCK(stcb);
 		hold_tcblock = 0;
 	} else {
-		atomic_add_int(&stcb->asoc.sb_send_resv, sndlen);
+		atomic_add_int(&stcb->asoc.sb_send_resv, (int32_t)sndlen);
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/078bd24ac1768bc6dc87d40b030434cba1987ea3.